packages/frontend/src/components/MkNoteDetailed.vue

```diff
@@ -392,29 +395,26 @@ if (appearNote.value.reactionAcceptance === 'likeOnly') {
 	});
 }
 
-function renote(viaKeyboard = false) {
+async function renote() {
```

Comment:
```suggestion
function renote() {
```
getRenoteMenuが非同期関数なのはtaiy fork固有なので、ここは戻して良いかも